src/client/app/actions/admin.ts

```diff
@@ -129,3 +129,20 @@ export function submitPreferencesIfNeeded(): Thunk {
 		return Promise.resolve();
 	};
 }
+
+function updateCikAndDBViews(): t.UpdateCikAndDBViews {
+	return { type: ActionType.UpdateCikAndDBViews };
+}
+
+/**
+ * Redo Cik and/or refresh reading views.
+ * This function is called when some changes in units/conversions affect the Cik table or reading views.
+ */
+export function updateCikAndDBViewsIfNeeded(shouldRedoCik: boolean, shouldRefreshReadingViews: boolean): Thunk {
```

Comment:
I didn't check the state inside this function since the loading page already prevents this function from being called multiple times. In other words, it's always called when the state is false. But to make it fit with naming standards, I put the check back.